packages/trader/src/AppV2/Components/TradeParameters/PayoutPerPoint/payout-per-point-wheel.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import debounce from 'lodash.debounce';
+import { ActionSheet, Text, WheelPicker } from '@deriv-com/quill-ui';
+import { Skeleton } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import type { TV2ParamsInitialValues } from 'Stores/Modules/Trading/trade-store';
+
+type TPayoutPerPointWheelProps = {
+    barrier?: string | number;
+    current_payout_per_point: string;
+    onPayoutPerPointSelect: (new_value: string | number) => void;
+    payout_per_point_list: {
+        value: string;
+    }[];
+    setV2ParamsInitialValues: ({ value, name }: { value: number | string; name: keyof TV2ParamsInitialValues }) => void;
+};
+
+const onWheelPickerScrollDebounced = debounce(
```

Comment:
Why are we debouncing? is there a usecase?

Reply:
We are debouncing, because this function is passed to WheelPicker and we don't won't to send the new value to BE couple times per second

Reply:
Same situation with GrowthRatePicker, MultiplierWheelPicker, TakeProfitAndStopLossInput, StrikeWheel component